tests/lib/helpers.js

```diff
@@ -10,7 +10,6 @@ var config = require('../lib/config');
 function done() {
     casper.options.pageSettings.userAgent = config.userAgent.phantomjs;
     casper.options.viewportSize = config.viewport.desktop;
-    casper.options.waitTimeout = 5000;
```

Comment:
Indeed, good catch. For some reason my mind was thinking, meh, we'll never need a longer timeout ;p